### PR TITLE
[WIP]: HTML bundle loader dynamically through import statement. (#178)

### PR DESCRIPTION
**Pull request by abiduzz420**
- [x] All tests passing.
- [x] Prettier format done

### Changes included:
- added `@webcomponents/webcomponentsjs` dependency as a polyfill
- created `loadHTMLBundle` method in `bundle-loader.js`
- included html-import directory in test/integrations.
- wrote a test to support dynamic imports in HTML

_This PR is still Work in Progress_

### CODE REVIEW

**Comment by abiduzz420** [comment]
This issue solves the problem from issue #178

**Comment by abiduzz420** [comment]
To-do
- [ ] inject webcomponents script as a dependency for the assets that need it
- [ ] Changes to be added to `HTMLPackager.js`

File: package.json [hunk 1]
```diff
@@ -8,6 +8,7 @@
     "url": "https://github.com/parcel-bundler/parcel.git"
   },
   "dependencies": {
+    "@webcomponents/webcomponentsjs": "^1.0.20",
```
Comment (devongovett) [inline package.json]: This shouldn't be a dependency of parcel - it should be in the project

**Comment by devongovett** [comment]
This will become easier after #473 is merged. It will allow you to register custom bundle loaders which will be included as needed.

**Comment by tan9** [comment]
Hi @abiduzz420 , any progress on this PR? I found #473 has been merged.

**Comment by devongovett** [comment]
Closing in favor of #1528.

**Pull request closed without merging**